recipe/meta.yaml

```
@@ -258,11 +265,11 @@ outputs:
       files:
         - test-r-xgboost.r
       commands:
-        - Rscript test-r-xgboost.r
+        - Rscript test-r-xgboost.r  # [cuda_compiler == "None" or not win]
```

Think this is equivalent. This just makes it a bit clearer that the issue is Windows CUDA tests (likely due to the lack of GPU)

```suggestion
        - Rscript test-r-xgboost.r  # [not (win and cuda_compiler != "None")]
```